fingerprintApis.mjs

```diff
@@ -1,11 +1,12 @@
 import _ from 'lodash';
+
 const weightsUrl = `https://raw.githubusercontent.com/duckduckgo/tracker-radar/refs/heads/main/build-data/generated/api_fingerprint_weights.json`;
```

Comment:
Is the idea here to fetch the latest value every time `gulp update-codeql` runs to generate new files? I was wondering if we should use a commit hash instead of the branch name for consistency.

eg: `https://raw.githubusercontent.com/duckduckgo/tracker-radar/ef963267a0c42f3d0aa83e8fb6874b3cec183a1e/build-data/generated/api_fingerprint_weights.json`